alpha/declcfg/load.go

```diff
@@ -168,3 +168,16 @@ func readYAMLOrJSON(r io.Reader) (*DeclarativeConfig, error) {
 	}
 	return cfg, nil
 }
+
+func LoadFile(root fs.FS, file io.Reader, path string) (*DeclarativeConfig, error) {
+	cfg, err := readYAMLOrJSON(file)
+	if err != nil {
+		return nil, err
+	}
+
+	if err = readBundleObjects(cfg.Bundles, root, path); err != nil {
+		return cfg, err
```

Comment:
Return nil here instead of a partial cfg?
```suggestion
		return nil, err
```